pgx/src/spi.rs

```diff
@@ -36,6 +36,8 @@ pub enum SpiOk {
     RelRegister = 15,
     RelUnregister = 16,
     TdRegister = 17,
+    #[cfg(feature = "pg15")]
+    Merge = 18,
```

Comment:
This is the wrong approach, as it requires users to `cfg` around it too, instead of letting them using one codebase for different versions of Postgres.

Reply:
This is worst in migration mode.